interface-definitions/netns.xml.in

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<interfaceDefinition>
+  <node name="netns" owner="${vyos_conf_scripts_dir}/netns.py">
+    <properties>
+      <help>Network namespace</help>
+      <priority>299</priority>
+    </properties>
+    <children>
+      <tagNode name="name">
+        <properties>
+          <help>Network namespace name</help>
+        </properties>
+        <children>
+          <leafNode name="descriprion">
```

Comment:
Besides this typo, please use the generic description XML building block.

Reply:
Done